eventservice/events.yaml

```
@@ -872,6 +872,16 @@ components:
           type: string
           format: uuid
           description: For an event created by a Rokwire group, this field stores the group ID.
+        groupIds:
+          type: array
+          description: The groupIds is a list of all groups that a user event is associated with.
```

```suggestion
          description: A list of IDs of all groups with administrative permissions on this event.
```
It may be better to specify the permissions here. Thanks.